src/main/java/com/wyden/findyourhome/entities/enums/KindPerson.java

```diff
@@ -0,0 +1,6 @@
+package com.wyden.findyourhome.entities.enums;
```

Comment:
Você tá pensando aplicar alguma regra de negócio aqui? Pensando no MVP, acho que quanto mais regras, maior complexidade.
Caso não tenha um tratamento específico, talvez o campo não seja necessário

Reply:
Eu não pensei em aplica regras de negócio, eu só pensei em registrar mais informações sobre o usuário para nossa aplicação se parecer mais com as que temos no mercado, pois é algo comum em alguns sites de anúncio o pedido de cpf/cnpj para registro. A única coisa que pensei em aplicar para esse campo é um validator para validar o cnpj ou cpf do ser humano.

Reply:
Então, eu acho uma complexidade a mais. Mas se for colocar o validador de CPF/CNPJ, acho que faz sentido manter.
Lembrando que seriam 2 validadores diferentes
E, se for manter, também precisaria cuidar para não exibir essa informação na resposta do enpoint, só

Reply:
tranquilo, deve ter biblioteca pronta para isso, se não tiver o gpt faz

Reply:
Certamente, tem. Se não tiver e for usar algum gpt pra implementar, importante testar para garantir que a validação adequada vai ser aplicada pra CPF e CNPJ